ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2857,6 +2857,16 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-zones
+- as: vsphere-ipi-ipi-remote-worker-p2-f13
```

Comment:
1. `f13`? not `f14`?
2. The ci jobs in the yaml file are placed in alphabetical order, so pls correct it.